specs/CalendarView/CalendarViewSpec1.md

```diff
@@ -0,0 +1,716 @@
+
+## CalendarView theme resources
+
+You can modify the look of a CalendarView by specifying Xaml resources in your app.
+For more info, see the
+[lightweight styling guide](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-styles#lightweight-styling).
+
+The new properties in this spec have default values according to new resources:
+
+| Property | Resource key
+| - | - |
+| BlackoutBackground | CalendarViewBlackoutBackground |
+| BlackoutStrikethroughBrush | CalendarViewBlackoutStrikethroughBrush |
+| CalendarItemDisabledBackground | CalendarViewCalendarItemDisabledBackground |
+| CalendarItemHoverBackground | CalendarViewCalendarItemHoverBackground |
+| CalendarItemPressedBackground | CalendarViewCalendarItemPressedBackground |
+| DayItemMargin | CalendarViewDayItemMargin |
+| DisabledForeground | CalendarViewDisabledForeground |
+| FirstOfMonthLabelMargin | CalendarViewFirstOfMonthLabelMargin |
+| FirstOfYearDecadeLabelMargin | CalendarViewFirstOfYearDecadeLabelMargin |
+| MonthYearItemMargin | CalendarViewMonthYearItemMargin |
+| OutOfScopeHoverForeground | CalendarViewOutOfScopeHoverForeground |
+| OutOfScopePressedForeground | CalendarViewOutOfScopePressedForeground |
+| SelectedDisabledBorderBrush | CalendarViewSelectedDisabledBorderBrush |
+| SelectedDisabledForeground | CalendarViewSelectedDisabledForeground |
+| SelectedHoverForeground | CalendarViewSelectedHoverForeground |
+| SelectedPressedForeground | CalendarViewSelectedPressedForeground |
+| TodayBackground | CalendarViewTodayBackground |
+| TodayBlackoutBackground | CalendarViewTodayBlackoutBackground |
+| TodayBlackoutForeground | CalendarViewTodayBlackoutForeground |
+| TodayDisabledBackground | CalendarViewTodayDisabledBackground |
+| TodayHoverBackground | CalendarViewTodayHoverBackground |
+| TodayPressedBackground | CalendarViewTodayPressedBackground |
+| TodaySelectedInnerBorderBrush | CalendarViewTodaySelectedInnerBorderBrush |
+
+
+# API Pages
+
+_Spec note: the following remarks apply to each of the new properties_
+
+This property is only used by the `CalendarView` control when a boolean resource named `CalendarViewBaseItemRoundedChromeEnabled` is set to True in the
+[Application.Resources](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Application.Resources).
+
+```xml
+    <x:Boolean x:Key="CalendarViewBaseItemRoundedChromeEnabled">True</x:Boolean>
+```
+
+This property is used by the
+[WinUI2 controls library](https://docs.microsoft.com/en-us/windows/apps/winui/winui2/),
+which sets this boolean resource to enable it.
+So using WinUI2 in your app causes this property to be enabled.
+
+## CalendarView.DayItemMargin
+
+`public Windows.UI.Xaml.Thickness DayItemMargin { get; set; }`
+
+Gets or sets the margin applied to the main label inside a calendar day item.
+
+This property is only used by the CalendarView control when a boolean theme resource named `CalendarViewBaseItemRoundedChromeEnabled` is set to True.
```

Comment:
Discussion: the dominant characteristic is the rounding. The margin properties are to enable the rounding. There are colors here too though.

Since rounding is the dominant characteristic, we'll use this name. (No obvious alternative.)